projects/opendr_ws/src/perception/scripts/object_tracking_2d_fair_mot.py

```diff
@@ -166,27 +171,53 @@ def draw_predictions(frame, predictions: TrackingAnnotation, is_centered=False,
         )
 
 
-if __name__ == "__main__":
-    # Automatically run on GPU/CPU
-    device = "cuda:0" if torch.cuda.is_available() else "cpu"
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-n", "--model_name", help="Name of the trained model",
+                        type=str, default="fairmot_dla34")
+    parser.add_argument("-t", "--temp_dir", help="Path to a temporary directory with models",
+                        type=str, default="temp")
+    parser.add_argument("-i", "--input_image_topic",
+                        help="Input Image topic provided by either an image_dataset_node, webcam or any other image node",
+                        type=str, default="/opendr/dataset_image")
+    parser.add_argument("-od", "--output_detection_topic",
+                        help="Output detections topic",
+                        type=str, default="/opendr/fairmot_detection")
+    parser.add_argument("-ot", "--output_tracking_id_topic",
+                        help="Output detections topic",
+                        type=str, default="/opendr/fairmot_tracking_id")
+    parser.add_argument("-oi", "--output_image_topic",
+                        help="Output detections topic",
```

Comment:
It would be helpful to add a more descriptive help text, right now it's the same as the detections topic, the docstring text from the class ctor would be ok.